src/india_api/internal/models.py

```diff
@@ -52,9 +52,39 @@ def to_timezone(self, tz: dt.timezone) -> "ActualPower":
         )
 
 
-class Site(BaseModel):
+class SiteProperties(BaseModel):
     """Site metadata"""
 
+    client_site_name: Optional[str] = Field(
+        None,
+        json_schema_extra={"description": "The name of the site as given by the providing user."},
+    )
+    orientation: Optional[float] = Field(
+        None,
+        json_schema_extra={
+            "description": "The rotation of the panel in degrees. 180° points south"
+        },
+    )
+    tilt: Optional[float] = Field(
+        None,
+        json_schema_extra={
+            "description": "The tile of the panel in degrees. 90° indicates the panel is vertical."
+        },
+    )
+    latitude: Optional[float] = Field(
+        None, json_schema_extra={"description": "The site's latitude"}, ge=-90, le=90
+    )
+    longitude: Optional[float] = Field(
+        None, json_schema_extra={"description": "The site's longitude"}, ge=-180, le=180
+    )
+    capacity_kw: Optional[float] = Field(
+        None, json_schema_extra={"description": "The site's total capacity in kw"}, ge=0
+    )
+
+
```

Comment:
I could have combiend with Site, but wanted to make everything optional so I thought this was quite a good simple layout